gluten-substrait/src/main/scala/org/apache/gluten/execution/BatchScanExecTransformer.scala

```diff
@@ -95,7 +95,12 @@ abstract class BatchScanExecTransformerBase(
 
   // Note: "metrics" is made transient to avoid sending driver-side metrics to tasks.
   @transient override lazy val metrics: Map[String, SQLMetric] =
-    BackendsApiManager.getMetricsApiInstance.genBatchScanTransformerMetrics(sparkContext)
+    BackendsApiManager.getMetricsApiInstance.genBatchScanTransformerMetrics(
+      sparkContext) ++ customMetrics
+
+  def doPostDriverMetrics(): Unit = {
```

Comment:
Why add the function instead of directly call `postDriverMetrics`

Reply:
because Symbol postDriverMetrics is inaccessible from this place